package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/fastify/fastify-express#readme",
   "devDependencies": {
+    "@types/express": "*",
```

Comment:
Can you add a note to the readme about users needing to install @types/express in order to use this module with typescript?

Reply:
sure! should I add an optional peer dep as well?

Reply:
We don't really use peerDeps in fastify - we've settled on just listing typescript deps in readmes/docs cause usually they are quite sparse

Reply:
Cool, done 🙂 (did the same for the `middie` PR)